service/src/main/kotlin/fi/espoo/evaka/application/notes/ApplicationNoteQueries.kt

```diff
@@ -127,3 +127,12 @@ fun Database.Transaction.deleteApplicationNote(id: ApplicationNoteId) {
 
     createUpdate(sql).bind("id", id).execute()
 }
+
+fun Database.Transaction.deleteApplicationNotesLinkedToMessages(
+    messageContentIds: List<MessageContentId>
```

Comment:
I would make the parameter `Set<MessageContentId>` because there can be duplicates (in theory).